farcasterapi/neynar/neynar.go

```diff
@@ -552,16 +583,19 @@ func (n *NeynarAPI) parseCastData(data *castWebhookData) (*farcasterapi.APIMessa
 }
 
 func (n *NeynarAPI) request(ctx context.Context, url, method string, body []byte, timeout time.Duration) ([]byte, error) {
-	for attempt := 0; attempt < maxRetries; attempt++ {
```

Comment:
why?

Reply:
i created another function to make and retry the request.